scripts/run_unit_tests.sh

```diff
@@ -3,14 +3,18 @@
 source $HOME/.cargo/env
 
 BUILD_TARGET=${BUILD_TARGET-x86_64-unknown-linux-gnu}
+cargo_args=("$@")
+[ $(uname -m) = "aarch64" ] && cargo_args+=("--no-default-features")
+[ $(uname -m) = "aarch64" ] && cargo_args+=("--features mmio")
+[ $(uname -m) = "aarch64" ] && cargo_args+=("--exclude pci")
```

Comment:
This line is not needed. By selecting `--no-default-features -- features mmio`, `pci` won't be part of the build.

Reply:
Yes I have expected that too, but I found it very weird that if I don't include `--exclude pci`, when I run `./script/dev_cli.sh tests --unit`, the pci module will still be compiled and give me this (I think this is caused by the refactor of the `SystemAllocator` as last time I didn't run into this):

```rust
[Cloud Hypervisor] Running unit tests for aarch64-unknown-linux-gnu...
   Compiling cloud-hypervisor v0.8.0 (/cloud-hypervisor)
   Compiling pci v0.1.0 (/cloud-hypervisor/pci)
error[E0599]: no method named `allocate_io_addresses` found for mutable reference `&mut vfio::vm_allocator::SystemAllocator` in the current scope
   --> pci/src/vfio.rs:763:22
    |
763 |                     .allocate_io_addresses(None, region_size, Some(0x4))
    |                      ^^^^^^^^^^^^^^^^^^^^^ help: there is a method with a similar name: `allocate_mmio_addresses`

error[E0599]: no method named `free_io_addresses` found for mutable reference `&mut vfio::vm_allocator::SystemAllocator` in the current scope
   --> pci/src/vfio.rs:874:31
    |
874 |                     allocator.free_io_addresses(region.start, region.length);
    |                               ^^^^^^^^^^^^^^^^^ help: there is a method with a similar name: `free_mmio_addresses`

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0599`.
error: could not compile `pci`.

To learn more, run the command again with --verbose.
```

Reply:
Do you think you can fix the root cause in the `SystemAllocator`?

Reply:
@sboeuf Yes of course.

Reply:
Seem that we need to distinguish the x86_64 and AArch64 cases in `pci/src/vfio.rs`. I have updated the code.